wiz-admission-controller/templates/opawebhook.yaml

```diff
@@ -87,6 +87,45 @@ webhooks:
 {{- end }}
 {{- if and (or .Values.opaWebhook.enabled .Values.imageIntegrityWebhook.enabled) (not $useCertManagerCerts) }}
 ---
+{{ if .Values.debugWebhook.enabled -}}
```

Comment:
Why this section is under `{{- if and (or .Values.opaWebhook.enabled .Values.imageIntegrityWebhook.enabled) (not $useCertManagerCerts) }}` statement?

Reply:
Also I think you need to update {{- if and (or .Values.opaWebhook.enabled .Values.imageIntegrityWebhook.enabled) (not $useCertManagerCerts) }} to also check dumper enabled